src/components/CodeView/index.spec.tsx

```diff
@@ -634,7 +672,7 @@ describe(__filename, () => {
     it('does not break when the element is null', () => {
       const element = null;
 
-      scrollToSelectedLine(element);
+      expect(() => scrollToSelectedLine(element)).not.toThrow();
```

Comment:
Note that this was a drive-by fix for a linter warning that I figured I'd make while I was in the file.

Reply:
We have a bunch of tests where we just want to make sure that something doesn't throw (because a bug was found where it did throw, and then we fixed that bug). At some point recently our linter started complaining about any tests that do not include any explicit assertions, so these tests now generate warnings. That's why we need to wrap them in an `expect` with `not.toThrow()`.